### PR TITLE
Re-add Small RTO Packs

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Re-adds Small RTO Packs, Req has them in the Misc section, and its also in the squad req vendors, though in much smaller quantity.

## Why It's Good For The Game

It's cool and neat.

## Changelog

:cl: Geeves
add: Re-adds Small RTO Packs, Req has them in the Misc section, and its also in the squad req vendors, though in much smaller quantity.
/:cl:

### CODE REVIEW

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 1]
```diff
@@ -123,6 +123,7 @@
 		list("Binoculars", round(scale * 2), /obj/item/device/binoculars, VENDOR_ITEM_REGULAR),
 		list("Rangefinder", round(scale * 1), /obj/item/device/binoculars/range, VENDOR_ITEM_REGULAR),
 		list("Laser Designator", round(scale * 1), /obj/item/device/binoculars/range/designator, VENDOR_ITEM_REGULAR),
+		list("Small Radio Telephone Pack", round(scale * 4), /obj/item/storage/backpack/marine/satchel/rto/small, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/requisitions.dm:126]: put them in the backpacks section

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 2]
```diff
@@ -222,6 +223,7 @@
 		list("M89-S Signal Flare Pack", round(scale * 1), /obj/item/storage/box/m94/signal, VENDOR_ITEM_REGULAR),
 		list("Machete Scabbard (Full)", round(scale * 5), /obj/item/storage/large_holster/machete/full, VENDOR_ITEM_REGULAR),
 		list("Binoculars", round(scale * 1), /obj/item/device/binoculars, VENDOR_ITEM_REGULAR),
+		list("Small Radio Telephone Pack", round(scale * 1), /obj/item/storage/backpack/marine/satchel/rto/small, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/requisitions.dm:226]: ditto

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Comment by Hatterhat** [comment]
shitty little radio backpack my beloved

**Review by stanalbatross — APPROVED** [review]

**Review by fira — APPROVED** [review]

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 3]
```diff
@@ -191,6 +192,7 @@
 		list("Shotgun Scabbard", round(scale * 5), /obj/item/storage/large_holster/m37, VENDOR_ITEM_REGULAR),
 		list("USCM Technician Welderpack", round(scale * 2), /obj/item/storage/backpack/marine/engineerpack, VENDOR_ITEM_REGULAR),
 		list("Technician Welder-Satchel", round(scale * 3), /obj/item/storage/backpack/marine/engineerpack/satchel, VENDOR_ITEM_REGULAR),
+		list("Small Radio Telephone Pack", round(scale * 1), /obj/item/storage/backpack/marine/satchel/rto/small, VENDOR_ITEM_REGULAR),
```
Comment (Nanu308) [inline code/game/machinery/vending/vendor_types/requisitions.dm:195]: Main Req only

**Review by Nanu308 — CHANGES_REQUESTED** [review]

**Review by Nanu308 — APPROVED** [review]

**Pull request merged**